packages/@icon-magic/distribute/src/create-icon-template.ts

```diff
@@ -35,9 +35,7 @@ export async function createHbs(
 
     // xmldom and other dom substitutions (like jsdom) add ...attributes="" and
     // the string replace below is an ugly hack to remove the empty string
-    fs.writeFile(path.join(outputPath, `${iconName}.hbs`), serializeToString(xml).replace(/...attributes=\"\"/g, '...attributes'), (err) => {
-      if (err) throw err;
-    });
+    fs.writeFileSync(path.join(outputPath, `${iconName}.hbs`), serializeToString(xml).replace(/...attributes=\"\"/g, '...attributes'));
```

Comment:
are you fully rebased? this is part of ramitha's change: https://github.com/linkedinlabs/icon-magic/commit/c274ee1c5f003be704e4184acea2fcecb97854f0

Reply:
Ah well Chris helped me with "fixing" my git branch.  We rebased with master and fixed merge conflicts with master (that had Ramitha's changes) and this is what happened.  Not sure what to do next but I know this branch has all the latest.

Reply:
fixed with latest push